website/server/controllers/api-v3/tasks.js

```diff
@@ -674,9 +683,17 @@ api.removeTagFromTask = {
     if (validationErrors) throw validationErrors;
 
     let taskId = req.params.taskId;
-    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id, { userId: user._id });
+    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id);
 
-    if (!task) throw new NotFound(res.t('taskNotFound'));
+    if (!task) {
```

Comment:
No, I don't believe this is correct. This will now allow a check for challenge or group tasks that do not have a user._id. You can see the previous logic for how this worked with challenges.